gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/internal/ClouddriverService.groovy

```diff
@@ -286,4 +281,9 @@ interface ClouddriverService {
   List<Map> getRoles(@Path("cloudProvider") String cloudProvider,
                      @Path("account") String account,
                      @Path("region") String region)
+
+  @GET('/ecsClusters/{cloudProvider}/{account}/{region}')
```

Comment:
No need for {cloudProvider} as this is ecs-specific.  Unless AWS folks did the same for their aws cluster controller? @dkirillov

Reply:
Removed.